routes/project/location/load_balancer.rb

```diff
@@ -118,6 +123,15 @@ class Clover
         lb.incr_refresh_cert
       end
 
+      r.post "enable-cert" do
+        authorize("LoadBalancer:edit", lb)
+        cert_enabled = typecast_params.bool!("cert_enabled")
+        cert_enabled ? lb.enable_cert_server : lb.disable_cert_server
```

Comment:
I would do the same check here done when creating a load balancer, and only call these methods if the setting has changed.

Also, this line and the audit log should be wrapped in a transaction.